JetAnalyzers/bin/jet_response_fitter_x.cc

```diff
@@ -20,6 +20,8 @@
 #include <TVirtualFitter.h>
 #include <TMath.h>
 #include <TSpectrum.h>
```

Comment:
I meant TVirtualFitter.h, TMath.h, and TSpectrum.h. I realize that those were already there, but I figure that since you are editing things it might be easier for you.